Modules/_io/bufferedio.c

```diff
@@ -1449,8 +1449,8 @@ _io_BufferedReader___init___impl(buffered *self, PyObject *raw,
         return -1;
     _bufferedreader_reset_buf(self);
 
-    self->fast_closed_checks = (Py_TYPE(self) == &PyBufferedReader_Type &&
-                                Py_TYPE(raw) == &PyFileIO_Type);
+    self->fast_closed_checks = Py_IS_TYPE(self, &PyBufferedReader_Type) &&
+                               Py_IS_TYPE(raw, &PyFileIO_Type);
```

Comment:
```suggestion
    self->fast_closed_checks = (Py_IS_TYPE(self, &PyBufferedReader_Type) &&
                                Py_IS_TYPE(raw, &PyFileIO_Type));
```

Reply:
Should I make a new PR with these changes?